src/test/java/org/jenkinsci/plugins/androidsigning/compatibility/SignApksBuilderCompatibility_2_0_8_Test.java

```diff
@@ -20,14 +18,19 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 
 
-public class SignApksBuilderCompatibility_2_0_8_Test {
+@WithJenkins
+class SignApksBuilderCompatibility_2_0_8_Test {
+
+    private JenkinsRule testJenkins;
 
-    @Rule
-    public JenkinsRule testJenkins = new JenkinsRule();
+    @BeforeEach
+    void beforeEach(JenkinsRule rule) {
+        testJenkins = rule;
+    }
```

Comment:
This can be considered the standard-way of using `JenkinsRule` with JUnit Jupiter. There are other ways of doing it, but this one is usually the easiest and most versatile way.